editor/scene_tree_dock.cpp

```diff
@@ -2127,6 +2141,10 @@ void SceneTreeDock::_node_reparent(NodePath p_path, bool p_keep_global_xform) {
 	}
 
 	_do_reparent(new_parent, -1, nodes, p_keep_global_xform);
+
+	for (Node *E : full_selection) {
+		editor_selection->add_node(E);
+	}
```

Comment:
why is this needed? I'm wondering as this might make #91860 resurface

Reply:
In this case it is for consistency so that when you reparent using the right click option `Reparent` it also keep all the reparented nodes selected